alpha-build-machinery/make/targets/openshift/crd-schema-gen.mk

```diff
@@ -0,0 +1,36 @@
+CRD_SCHEMA_GEN_VERSION ?=v0.2.1
+CRD_SCHEMA_GEN_APIS ?=$(error CRD_SCHEMA_GEN_APIS is required) 
+CRD_SCHEMA_GEN_TEMP ?=_output/tools/src/sigs.k8s.io/controller-tools
+CRD_SCHEMA_GEN_OUTPUT ?=./manifests
+
+crd-schema-gen:
+	if [ ! -d $(CRD_SCHEMA_GEN_TEMP) ]; then \
+		mkdir -p $(CRD_SCHEMA_GEN_TEMP); \
+		git clone -b $(CRD_SCHEMA_GEN_VERSION) --single-branch --depth 1 https://github.com/kubernetes-sigs/controller-tools.git $(CRD_SCHEMA_GEN_TEMP); \
```

Comment:
can you reference the repo with the same url?

Reply:
You mean `sigs.k8s.io`? Doesn't work for me:
```
$ git clone sigs.k8s.io/controller-tools
fatal: repository 'sigs.k8s.io/controller-tools' does not exist
```

Reply:
yeah, looks like `git clone` doesn't handle 403